runtime-modules/service-discovery/src/mock.rs

```diff
@@ -167,6 +167,7 @@ pub(crate) fn hire_storage_provider() -> (u64, u64) {
         role_account_id,
         reward_relationship: None,
         role_stake_profile: None,
+        storage: Vec::new(),
```

Comment:
I think you forgot to remove this field

Reply:
https://github.com/Joystream/joystream/pull/2180/commits/dc39181954b99319c9fb98b0ff9cecb3ec475fe5